cmd/agent/subcommands/run/command.go

```diff
@@ -404,7 +408,7 @@ func startAgent(
 	telemetry telemetry.Component,
 	_ sysprobeconfig.Component,
 	server dogstatsdServer.Component,
-	serverDebug dogstatsddebug.Component,
+	_ dogstatsddebug.Component,
```

Comment:
Since this component is reference by the runtime settings component we no longer need to pass it as part of the `startAgent` function.